money_bot/utils/db_utils.py

```diff
@@ -1,6 +1,22 @@
 from money_bot.utils import models
 
 
+async def is_user_in_db(user_id: int):
+    return bool(await get_user_by_id(user_id))
+
+
+async def set_referrer_id(user_id: int, referrer_id: int):
```

Comment:
Is this function actually useful? I think this code will be used only once

Reply:
it's just to make code clearer